includes/Dashboard/Templates/Settings.php

```diff
@@ -193,71 +193,60 @@ public function load_store_content() {
      * @return void
      */
```

Comment:
Rewrite this method as below:
```
        $payment_methods = dokan_withdraw_get_active_methods();

        //no payment method is active, show informative message
        if ( empty( $payment_methods ) ) {
            dokan_get_template_part(
                'global/dokan-error',
                '',
                [ 'deleted' => false, 'message' => __( 'No withdraw method is available. Please contact site admin.', 'dokan-lite' ) ]
            );
            return;
        }

        $seller_id = dokan_get_current_user_id();

        // Find the payment methods
        $seller_connected_payment_method_ids = array_filter(
            $payment_methods,
            function ( $payment_method ) use ( $seller_id ) {
                return $this->is_seller_connected( $payment_method, $seller_id );
            }
        );

        $seller_disconnected_payment_method_ids = array_diff( $payment_methods, $seller_connected_payment_method_ids );
        $seller_disconnected_payment_methods    = $this->get_payment_methods( $seller_disconnected_payment_method_ids );
        $seller_connected_payment_methods       = $this->get_payment_methods( $seller_connected_payment_method_ids );

        /*
         * If we are requesting a single payment method page (to edit or for first time setup)
         * then we have the corresponding payment method key in the url.
         */
        $method_key   = str_replace( '/manage-', '', $slug_suffix );
        $is_edit_mode = false;

        /*
         * If payment method key has /edit suffix then we are trying to edit the method,
         * otherwise we are doing a initial setup for that payment method.
         */
        if ( false !== stripos( $method_key, '/edit' ) ) {
            $is_edit_mode = true;
            $method_key   = str_replace( '/edit', '', $method_key ); // removing `/edit` suffix to get payment method key
        }

        $profile_info = get_user_meta( $seller_id, 'dokan_profile_settings', true );

        if ( $is_edit_mode && 'bank' === $method_key ) {
            $profile_info['is_edit_mode'] = $is_edit_mode;
        }

        // Template arguments
        $args = [
            'current_user' => $seller_id,
            'profile_info' => $profile_info,
        ];

        if ( empty( $method_key ) ) { // payment method list page arguments
            $args = array_merge(
                $args,
                [
                    'methods'        => $seller_connected_payment_methods,
                    'unused_methods' => $seller_disconnected_payment_methods,
                ]
            );

            // Show the payment method list template
            dokan_get_template_part( 'settings/payment', '', $args );
            return;
        }

        // Get the single payment method for the $method_key
        $method = dokan_withdraw_get_method( $method_key );
        $args   = array_merge(
            $args,
            [
                'method'     => $method,
                'method_key' => $method_key,
            ]
        );

        if ( empty( $method ) || ! isset( $method['callback'] ) || ! is_callable( $method['callback'] ) ) {
            dokan_get_template_part(
                'global/dokan-error',
                '',
                [ 'deleted' => false, 'message' => __( 'Invalid withdraw method. Please contact site admin', 'dokan-lite' ) ]
            );
            return;
        }

        // Show the single payment method page otherwise
        dokan_get_template_part( 'settings/payment', 'manage', $args );
```